setup.py

```diff
@@ -19,8 +19,8 @@
 
 setup(
     name="trieste",
-    version="0.1.0",
-    author="The Trieste authors",
+    version="0.2.0",
```

Comment:
we're uploading to the already existent trieste pypi package. That has versions 0.1.0 and 0.1.1, so bumping to 0.2.0 clearly delineates this from the previous trieste project. We may choose to delete the 0.1.0 and 0.1.1 versions